css-layout-classes/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1>Choose your Character!</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-two-third">
+            <div class="column-one-second"><img src="images/link.png" /></div>
```

Comment:
This `column-one-second` class is unnecessary, and causing your images not to display propertly.

The `img` should just be in the `column-two-third` directly